pkg/profiler/cpu/bpf/maps/maps.go

```diff
@@ -134,7 +133,7 @@ const (
 	compactUnwindRowSizeBytesArm64           = 16
 	minRoundsBeforeRedoingUnwindInfo         = 5
 	minRoundsBeforeRedoingProcessInformation = 5
-	maxCachedProcesses                       = 10_0000
+	maxCachedProcesses                       = 100_000
```

Comment:
What led to the 10x increase in the number of cached processes?

Reply:
If you look carefully it's not an increase. I have moved the underscore to proper location.